indico/modules/events/abstracts/forms.py

```diff
@@ -0,0 +1,33 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2016 European Organization for Nuclear Research (CERN).
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 3 of the
+# License, or (at your option) any later version.
+#
+# Indico is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Indico; if not, see <http://www.gnu.org/licenses/>.
+
+from wtforms.fields import BooleanField, IntegerField
+from wtforms.validators import NumberRange, Optional
+
+from indico.util.i18n import _
+from indico.web.forms.base import IndicoForm
+from indico.web.forms.widgets import SwitchWidget
+
+
+class AbstractDescriptionSettingsForm(IndicoForm):
+    """Configure the description field of abstracts"""
+
+    is_active = BooleanField(_('Active'), widget=SwitchWidget(),
+                             description=_("Whether the description field is available."))
```

Comment:
i think we call it "content" in abstracts, but i'm not sure. maybe "description/content field" would fit better here

cc @pferreir